lib/gltfPipeline.js

```diff
@@ -53,17 +53,7 @@ function processJSONWithExtras(gltfWithExtras, options, callback) {
     convertDagToTree(gltfWithExtras);
     combineMeshes(gltfWithExtras);
     combinePrimitives(gltfWithExtras);
-
-    // run the AO stage
-    if (options.ao_diffuse) {
-        var aoOptions = {
-            scene: options.ao_scene,
-            rayDepth: options.ao_rayDepth,
-            numberSamples: options.ao_samples,
-            resolution: options.ao_resolution
-        };
-        bakeAmbientOcclusion(gltfWithExtras, aoOptions);
-    }
+    bakeAmbientOcclusion(gltfWithExtras, options.aoOptions);
```

Comment:
I think it's a little clearer and more in-line with the other stages if we check if we can run the ao stage before going into the function.

```javascript
if (defined(options.aoOptions) && options.aoOptions.runAO) {
    bakeAmbientOcclusion(gltfWithExtras, options.aoOptions);
}
```

And possibly just remove `runAO` altogether. As long as the ao options exist, it should be able to run the stage. This would require some edits in `gltf-pipeline.js`.